src/sql/workbench/contrib/charts/browser/actions.ts

```diff
@@ -106,6 +108,28 @@ export class CreateInsightAction extends Action {
 	}
 }
 
+export class ConfigureChartAction extends Action {
+	public static ID = 'chartview.configureChart';
+	public static LABEL = localize('configureChartLabel', "Configure Chart");
+	public static ICON = 'filterLabel';
```

Comment:
>filterLabel [](start = 23, length = 11)

Why are you using the filter icon here?

Reply:
The only gear icon I could find was the class **codicon-settings-gear**, which overrides some font and size styling.

Reply:
Couldn't you override those stylings yourself in the css for this?

If that's not possible for some reason we should make our own then. The icon as it currently stands is definitely not going to be something people expect/understand to mean settings.

Reply:
Looks like the icons are drawn from a custom font, so overriding the font is something we can't do. They probably aren't meant to be used for labels. I'll see if I can extract the settings icon somehow.

https://github.com/microsoft/azuredatastudio/blob/master/src/vs/base/browser/ui/codiconLabel/codicon/codicon.css#L8

Reply:
Added a new icon.